.env.test.local

```diff
@@ -0,0 +1,2 @@
+SUPABASE_URL=https://ezwbsacoojmonmiqffad.supabase.co
```

Comment:
Oops! Make sure all `.env` files are included in your `.gitignore` before pushing